app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -28,12 +45,23 @@ fun PlotSurface() {
         modifier = Modifier.fillMaxSize(),
         color = Color.White
     ) {
-        // TODO Build out the plot surface
-        //  This should include a Column composable that
-        //  includes a Map, and two MapSlider composables
-        //  (one slider for each axis).
+        var xPercentage: Float by remember { mutableStateOf(0.5f) }
```

Comment:
Please clear the warnings on lines 48 & 49

Prefer `mutableFloatStateOf` instead of `mutableStateOf`